core/views.py

```diff
@@ -144,7 +144,10 @@ def contribute(request):
 
 
 def donate(request):
-    return render(request, 'core/donate.html', {})
+    return render(request, 'core/donate.html', {
+        'country_count': Event.objects.values('country').distinct().count(),
```

Comment:
I think this line is unnecessary, as the same count is already done in `context_processors` so should be available in that template as well.